subsys/net/lib/openthread/platform/radio.c

```diff
@@ -272,6 +280,21 @@ void transmit_message(struct k_work *tx_job)
 	radio_api->set_channel(radio_dev, sTransmitFrame.mChannel);
 	radio_api->set_txpower(radio_dev, tx_power);
 
+#if defined(CONFIG_OPENTHREAD_TIME_SYNC)
```

Comment:
Hmm, is there a way to append IEs from the interrupt in `nrf_802154_tx_started` somehow?